src/stan/io/deserializer.hpp

```diff
@@ -525,8 +525,8 @@ class deserializer {
 
   /**
    * Return the next simplex of the specified size (using one fewer
-   * unconstrained scalars), incrementing the specified reference with the
-   * log absolute Jacobian determinant.
+   * unconstrained scalars). This particular transformation is linear, not
```

Comment:
this is on the `simplex_constrain` and it is not linear, possibly a copy paste error

Reply:
Good catch, definitely a copy/paste